src/js/reducers.js

```diff
@@ -0,0 +1,14 @@
+var store = Redux.createStore(bartData);
+
+function bartData(state = { apiData: {} }, action) {
+  switch (action.type) {
+    case "BART_DATA_UPDATE":
+      if (state.apiData === undefined) {
+        return state;
+      }
+      state.apiData = action.apiData;
+      return state;
```

Comment:
you should not return `state` but the actual state, something like:
```javascript
{ apiDate: payload.data } 
```

Reply:
@eladg fixed this in the new method in the new file (data.js)